test/sdk/csharp/Program.cs

```diff
@@ -0,0 +1,164 @@
+﻿using Agones;

```

Comment:
Apache licence please 😄

Reply:
Thank you. Fixed [2dfe70a](https://github.com/googleforgames/agones/pull/3392/commits/2dfe70a8d35692481a52ae627bfea804e8ce86a1)